controllers/component_build_controller_pipeline.go

```diff
@@ -484,25 +480,6 @@ func getContainerImageRepository(image string) string {
 	return strings.Split(image, ":")[0]
 }
 
-// getComponentImageRepoAndSecretNameFromImageAnnotation parses image.redhat.com/image annotation
-// for image repository and secret name to access it.
-// If image.redhat.com/image is not set, the procedure returns empty values.
-func getComponentImageRepoAndSecretNameFromImageAnnotation(component *appstudiov1alpha1.Component) (string, string, error) {
```

Comment:
Wondering, is this no longer used?

Reply:
That was a fallback, but that option is deleted with old "annotation" reconciler in [Image Controller](https://github.com/konflux-ci/image-controller).